### ISSUE TITLE
saved spam submissions can't be viewed

### ISSUE THREAD

**Issue by samuelbirch** [verbb/formie#616]
The saved spam submissions can no longer be view in the cp.

v1.4.22

**Comment by engram-design** [comment]
Not sure what you mean, sorry. Is there no "Spam" item in the status dropdown in the Submissions index?

**Comment by samuelbirch** [comment]
Yes that is correct, i no longer have the "spam" item in the status dropdown.

**Comment by samuelbirch** [comment]
the previous version that was installed was 1.4.18 (in case that helps in anyway)

**Comment by engram-design** [comment]
Odd. Is there an "Incomplete" option?

![image](https://user-images.githubusercontent.com/1221575/138843374-d1f5d43a-2e24-4b89-81ab-3c6d35e83b3c.png)

**Comment by samuelbirch** [comment]
no, only the statuses and "trashed"
![Screenshot 2021-10-26 at 09 46 55](https://user-images.githubusercontent.com/13586/138843794-d03edec8-dd62-4f22-8977-1945c390c855.png)

**Comment by samuelbirch** [comment]
interesting, its only in "dev" that these don't appear. in the "staging" and "production" its fine

**Comment by engram-design** [comment]
Are there any JS errors? Can you confirm if the `formie-cp.js` resource is loaded in the Network tab of developer tools?

Wondering if you need to clear the caches locally?

**Comment by samuelbirch** [comment]
![Screenshot 2021-10-26 at 09 52 52](https://user-images.githubusercontent.com/13586/138844799-85d6c56f-7745-4247-9028-ba2f065ce2cc.png)
![Screenshot 2021-10-26 at 09 53 29](https://user-images.githubusercontent.com/13586/138844921-d431425b-b361-484d-a2b5-0d415ef9162a.png)

**Comment by engram-design** [comment]
So that `Craft.Formie.editableSubmissions` variable is defined in the Submissions index template. I'm wondering if you can see that JS on the page load, or if clearing the cache is required? 

https://github.com/verbb/formie/blob/craft-3/src/templates/submissions/index.html#L19

Otherwise, I've added some guards to prevent this unrelated error from messing up the rest of the JS. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.22",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.23](https://github.com/verbb/formie/releases/tag/1.4.23)
